loadtest/control/simulcontroller/controller_test.go

```diff
@@ -26,31 +28,25 @@ func TestNew(t *testing.T) {
 		WebSocketURL: "ws://localhost:8065",
 	})
 
-	c, err := New(1, user, config, make(chan control.UserStatus))
-	require.Nil(t, err)
-
-	require.Equal(t, len(c.actionList), len(c.actionMap))
-}
+	statusChan := make(chan control.UserStatus)
 
-func TestSetRate(t *testing.T) {
-	config, err := ReadConfig("../../../config/simulcontroller.sample.json")
+	c, err := New(1, user, config, statusChan)
 	require.NoError(t, err)
-	require.NotNil(t, config)
 
-	store, err := memstore.New(nil)
-	require.NotNil(t, store)
-	require.NoError(t, err)
+	return c, statusChan
+}
 
-	user := userentity.New(userentity.Setup{Store: store}, userentity.Config{
-		ServerURL:    "http://localhost:8065",
-		WebSocketURL: "ws://localhost:8065",
-	})
+func TestNew(t *testing.T) {
+	c, _ := newController(t)
```

Comment:
what if we just close the chan after this :p

Reply:
Why not!

Reply:
Done in https://github.com/mattermost/mattermost-load-test-ng/pull/565/commits/a0205a8a75f9449b011ff17837c00a8a007f6832